Source/Generic/ECTestComparisons.h

```diff
@@ -6,21 +6,24 @@
 
 #import <Foundation/Foundation.h>
 
-
-typedef NS_ENUM(NSUInteger, ECTestComparisonMode)
+// Property list, dictionary, etc. test comparison options
+typedef NS_OPTIONS(NSUInteger, ECTestComparisonOptions)
 {
-	ECTestComparisonShowChars,
-	ECTestComparisonShowLines,
-	ECTestComparisonShowLinesIgnoreWhitespace,
-	ECTestComparisonDiff,
-	ECTestComparisonDiffNoJSON,
-	ECTestComparisonDefault
+	ECTestComparisonNone = 0,
+	
+	// If this option is set then any NSNumber double values will be compared with a fudge factor.
+	ECTestComparisonDoubleFuzzy = 0x0001
```

Comment:
OK, so why's this enum being split across two files now? (I think that's what's happening?)